src/e2e-test/test-rule-functions.js

```diff
@@ -0,0 +1,56 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+var testUtils = require('./testUtils');
+var fhirModule = require('fhir');
+var fhir = new fhirModule.Fhir();
+
+var response = function(status, message=''){
+    return { valid: status, errorMessage: message};
+};
+
+var fhirR4Validation =  function(resJson){
+    var result = fhir.validate(resJson);
+    if (!result.valid)
+        return response(false, JSON.stringify(result, null, "\t"));
+    return response(true);
+};
+
+var onePatient = function(resJson){
+    var resources = fhir.evaluate(resJson, 'Bundle.entry.resource.resourceType');
+    var patientCount = testUtils.countOccurences(resources,'Patient');
+    if( patientCount !== 1)
+        return response(false, 'The bundle contains ' + patientCount + ' Patient resources');
+    else
+        return response(true);
+};
+
+var noDefaultGuid = function(resJson){
+    var ids = fhir.evaluate(resJson, 'Bundle.entry.resource.id');
+    var defaultGuidCount = testUtils.countOccurences(ids, testUtils.defaultGuid);
+    if(defaultGuidCount === 1)
+        return response(false, 'The bundle contains ' + defaultGuidCount + ' default Guid ' + testUtils.defaultGuid);
+    else if(defaultGuidCount > 1)
+        return response(false, 'The bundle contains ' + defaultGuidCount + ' default Guids ' + testUtils.defaultGuid);
```

Comment:
why do we need to differentiate between 1 and > 1?

Reply:
As Robert said, Guid should be plural when >1.  (Guids)

Reply:
please write it as "(s)". We don't need if/else for this.

Reply:
OK, I have modified it.